orchestration/_tests/test_globus_flow.py

```diff
@@ -0,0 +1,154 @@
+        time.sleep(polling_interval)
+        return True
+
+    def get_task(self, task_id):
+        return {"task_id": task_id, "status": "SUCCEEDED"}
+
+
+class FailedTransferClient(MockTransferClient):
+    def get_task(self, task_id):
+        return {"task_id": task_id, "status": "FAILED"}
+
+
+def test_globus_config():
+    config_file = Path(__file__).parent / "test_config.yml"
+    with MonkeyPatch.context() as mp:
+        mp.setenv("TEST_USERNAME", "TEST_USERNAME")
+        mp.setenv("TEST_PASSWORD", "TEST_PASSWORD")
+        globus_config = read_config(config_file=config_file)
+        endpoints = build_endpoints(globus_config)
+        test_endpoint = endpoints["test_endpoint"]
+
+        assert test_endpoint.root_path == "/data"
+        assert test_endpoint.uri == "test.example"
+        assert test_endpoint.uuid == "12345"
+
+
+def test_succeeded_transfer():
+    transfer_client = MockTransferClient()
+    source_endpoint = GlobusEndpoint("123", "source.magrathea.com", "/root")
+    dest_endpoint = GlobusEndpoint("456", "dest.magrathea.com", "/root")
+
+    result = start_transfer(
+        transfer_client, source_endpoint, "/42/mice.jpg", dest_endpoint, "/42/mice.jpg"
+    )
+
+    assert result
+
+
+def test_failed_transfer():
+    transfer_client = FailedTransferClient()
+    source_endpoint = GlobusEndpoint("123", "source.magrathea.com", "/root")
+    dest_endpoint = GlobusEndpoint("456", "dest.magrathea.com", "/root")
+
+    result = start_transfer(
+        transfer_client, source_endpoint, "/42/mice.jpg", dest_endpoint, "/42/mice.jpg"
+    )
+
+    assert result
+
+
+def test_transfer_data_to_alcf():
+    transfer_client = MockTransferClient()
+    source_endpoint = GlobusEndpoint("123", "source.magrathea.com", "/root")
+    dest_endpoint = GlobusEndpoint("456", "dest.magrathea.com", "/root")
+    file_path = "/path/to/file.txt"
+    result = transfer_data_to_alcf(file_path, transfer_client, source_endpoint, dest_endpoint)
+    assert result
+
+
+def test_transfer_data_to_nersc():
+    transfer_client = MockTransferClient()
+    source_endpoint = GlobusEndpoint("123", "source.magrathea.com", "/root")
+    dest_endpoint = GlobusEndpoint("456", "dest.magrathea.com", "/root")
+    file_path = "/path/to/file.txt"
+    result = transfer_data_to_nersc(file_path, transfer_client, source_endpoint, dest_endpoint)
+    assert result
+
+
+def test_transfer_data_to_data832():
+    transfer_client = MockTransferClient()
+    source_endpoint = GlobusEndpoint("123", "source.magrathea.com", "/root")
+    dest_endpoint = GlobusEndpoint("456", "dest.magrathea.com", "/root")
+    file_path = "/path/to/file.txt"
+    result = transfer_data_to_data832(file_path, transfer_client, source_endpoint, dest_endpoint)
+    assert result
+
+
+def test_schedule_prune_task():
+    path = "/path/to/prune"
+    location = "test_location"
+    schedule_days = datetime.timedelta(days=0.01)
+    result = schedule_prune_task(path, location, schedule_days)
```

Comment:
You're running some code, but maybe not testing it? It seems to me that success here would be tested by using the prefect api to test if the pruning task was succesfully scheduled. This feels like it belongs in a new `test_prefect.py` module.